planning/obstacle_stop_planner/include/obstacle_stop_planner/node.hpp

```diff
@@ -91,6 +92,17 @@ using vehicle_info_util::VehicleInfo;
 using TrajectoryPoints = std::vector<TrajectoryPoint>;
 using PointCloud = pcl::PointCloud<pcl::PointXYZ>;
 
+struct ObstacleWithDetectionTime
+{
+  ObstacleWithDetectionTime() = default;
+  ObstacleWithDetectionTime(const rclcpp::Time & t, pcl::PointXYZ & p) : detection_time(t), point(p)
```

Comment:
Do you have any reason why you use default?
The following would be better.
```suggestion
  explicit ObstacleWithDetectionTime(const rclcpp::Time & t, pcl::PointXYZ & p) : detection_time(t), point(p)
```

Reply:
Thanks for your advise. There is no clear reason :sweat_smile: I'll fix as you said.

Reply:
fixed in https://github.com/autowarefoundation/autoware.universe/pull/2647/commits/02b7c18c86ec056ec11856ddee140fc21653db68